src/test/requestCoordsTest.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
+
+const chai = require('chai');
+const Coords = require('../server/requests/requestCoords.js');
+
+const should = chai.should();
+let lat = '';
+let lon = '';
+
+describe('getLat', () => {
+  it('get Latitude', (done) => {
+    Coords.getCoords('brasilia').then((value) => {
+      lat = value.getLatitude();
+      should.equal(lat, '-10.3333333');
+      done();
+    });
+  });
```

Comment:
`.timeout(5000);`

De acordo com essa referência: https://stackoverflow.com/questions/15971167/how-to-increase-timeout-for-a-single-test-case-in-mocha